src/shaderlib/fp64/math-fp64.glsl

```diff
@@ -104,9 +104,11 @@ vec2 nint_fp64(vec2 a) {
 
 #if defined(NVIDIA_EQUATION_WORKAROUND) || defined(INTEL_EQUATION_WORKAROUND)
 vec2 twoSum(float a, float b) {
-  float s = (a + b) * ONE;
-  float v = (s - a);
-  float err = (a - (s - v) * ONE) * ONE + (b - v);
+  float s = (a + b);
+  float v = (s * ONE - a) * ONE;
+  float err = (a - (s - v) * ONE) * ONE * ONE * ONE + (b - v);
```

Comment:
I'll make the comments clearer. 

It's not trial and error. The principle here is to expand the final express to a polynomials of ONE and make each coefficient contains at most one a and one b.